addons/xeh/fnc_initDisplay.sqf

```diff
@@ -15,5 +15,5 @@ if !(_event isEqualTo "") then {
             private ["_event", "_className"]; // prevent these variables from being overwritten
             _args call compile getText _x;
         } forEach configProperties [_x >> XEH_FORMAT_CONFIG_NAME(_event) >> _className, "isText _x"];
-    } forEach XEH_MAIN_CONFIGS;
```

Comment:
why?

Reply:
Macro seemed pointless. Bugged me for a while and now is an opportunity to muck it out.